src/Cacher.ts

```diff
@@ -1,7 +1,12 @@
 import Manifest from "./Manifest";
 
 interface Cacher {
-    getManifest(manifestUrl: URL): Promise<Manifest>;
+    getManifest(manifestUrl: URL): Promise<Manifest>;   
```

Comment:
After discussing this with @aslagle, it seems like `getManifest` need not be part of `Cacher` functionality and can be moved out to another location.

I should also note that the `getManifest` method of `Cacher` does _not_ return a cache manifest as I first thought! It instead returns the manifest for the book that contains spine items and the like. Perhaps renaming this method to `getJSONManifest` or similar would make things more clear.